### ISSUE TITLE
Explotar grupo

### ISSUE THREAD

**Issue by etchegaray** [olvidio/tramity#29]
Cuando se hace un envío a un grupo y en la parte final, antes de enviar a secretaría, se hace un "explotar", la cosa funciona bien pero hasta un límite de ctr. Si se intenta hacer con el grupo "todos los ctr", se para en un determinado número y ya no es posible recuperarlo. Hemos probado también esperando por si se tratara de una operación que se está ejecutando en segundo plano, pero después de esperar mucho tiempo tampoco salen más ctr. También comprobamos mirando ('top' en la ventana de la aplicación) la tarea que se estaba ejecutando cuando se pulsa "explotar" y efectivamente en un momento dado se paraba.

No es algo que se utilice muchas veces, pero quizá se pueda intentar controlar ese error.

**Comment by Jordi59** [comment]
Sí. A nosotros también nos ha pasado en algunos "explotar" masivos. Se mejoró pero recientemente nos ha vuelto a pasar (explotar con más de 100 destinos). Pero alguna vez nos ha funcionado bien... Iría bien poder identificar la causa.

**Comment by olvidio** [comment]
Quizá fue al actualizar el php (ahora 8.3) He encontrado unos cambios apuntados que he vuelto a hacer para laconfiguración actual. A ver si con esto mejora:

/etc/php/8.3/fpm/pool.d/fpm-tramity.conf

    request_terminate_timeout = 300

/etc/php/8.3/fpm/php.ini

    max_execution_time = 300
